res/lazymc.toml

```diff
@@ -33,6 +33,10 @@ directory = "."
 # Warning: if using a bash script read: https://git.io/JMIKH
 command = "java -Xmx1G -Xms1G -jar server.jar --nogui"
 
+# Freeze the server process instead of restarting it when no players online, making it start up faster.
```

Comment:
Nitpick:

```diff
-# Freeze the server process instead of restarting it when no players online, making it start up faster.
+# Freeze the server process instead of restarting it when no players online, making it resume faster.
```

Reply:
Good catch, idk why I wrote "start up" when the process is just frozen